### DISCUSSION TITLE
How can I use Harness in own specs

### DISCUSSION THREAD

**Discussion by loxy** [taiga-family/taiga-ui#3118] [Q&A] [answered]
Hi!

Is it possible to use the Harness also in own unit tests. If yes, how do I import them (from where)?

Best regards,
Kersten

**Comment by splincode** [comment, marked as answer]
@waterplea should we publish them?

<img width="1029" alt="image" src="https://user-images.githubusercontent.com/12021443/204002207-8de20eb2-d17d-44fc-9b67-e9b7103de9fc.png">

**Reply by waterplea** [reply]
Yes, I think it's about time we publish them and continue developing them with publishing versions from now on.

**Comment by loxy** [comment]
Any news on this?

**Reply by loxy** [reply]
Ah, I see it was released first time 10 days ago, right?

**Reply by splincode** [reply]
yes
